source/configuration_manual/director_settings.rst

```diff
@@ -56,15 +56,7 @@ By default all active sessions will be shut down when director is reloaded or re
         user = dovecot
     }
    }
-   service imap-login {
-    executable = imap-login director
-   }
-   service pop3-login {
-    executable = pop3-login director
-   }
-   service managesieve-login {
-    executable = managesieve-login director
-   }
+   login_auth_socket_path = director
```

Comment:
Unlike the example-config, this doc.dovecot.org will be read by people with all kinds of random Dovecot versions. So unless it's clearly specified, people with wrong versions will most likely copy&paste the settings and wonder why it doesn't work. So these could be clarified like:

```
# v2.4+:
login_auth_socket_path = director
# Older versions:
service imap-login {
  executable = imap-login director
}
service pop3-login {
  executable = pop3-login director
}
service managesieve-login {
  executable = managesieve-login director
}
```